web_autoflip_demo/autoflip_demo_single_worker/main.ts

```diff
@@ -0,0 +1,137 @@
+/**
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+interface CropInfo {
+  type: string;
+  cropWindows: CropWindow[];
+  startTime: number;
+  videoId: number;
+  workerId: number;
+  shots: number[];
+}
+interface CropWindow {
+  width: number;
+  height: number;
+  x: number;
+  y: number;
+  time: number;
+}
+
+const myWorker = new Worker('worker.js');
+let videoBuffer = {} as ArrayBuffer;
+let videoInfo = {} as { duration: number; width: number; height: number };
```

Comment:
change to "let videoInfo: { duration: number; width: number; height: number }; = {}" It would be better if you could create an interface with the type.

Reply:
Fixed.